ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetLogs.java

```diff
@@ -41,16 +41,19 @@
   private final PrivacyQueries privacyQueries;
   private final PrivacyController privacyController;
   private final PrivacyIdProvider privacyIdProvider;
+  private final boolean multiTenancyEnabled;
 
   public PrivGetLogs(
       final BlockchainQueries blockchainQueries,
       final PrivacyQueries privacyQueries,
       final PrivacyController privacyController,
-      final PrivacyIdProvider privacyIdProvider) {
+      final PrivacyIdProvider privacyIdProvider,
+      final boolean multiTenancyEnabled) {
```

Comment:
Instead of pushing this extra flag through the call stack, couldn't we have a getter for this in the `PrivacyController`?

Reply:
Fixed. Uses the instance of method that you commented on and that we need to fix later ...